### ISSUE TITLE
Saving file does not work after windows update

### ISSUE THREAD

**Issue by om8000** [brandoncomputer/powershell-designer#5]
- Might be local settings, where we use (;) instead of a (,), or might be something in .NET ?

20221128 11:52:11: Cannot convert value "800; 600" to type "System.Int32". Error: "Input string was not in a correct format." At line:1024 char:6 + $n[0] = [math]::round(($n[0]/1) / $tscale) + ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~  + CategoryInfo : InvalidArgument: (:) [], RuntimeException  + FullyQualifiedErrorId : InvalidCastFromStringToInteger

**Comment by loial** [comment]
A single line of code fixed this issue for me (I created a PR #6 ), by adding `[Threading.Thread]::CurrentThread.CurrentCulture = 'en-US';` on the beginning of line 393 in Designer.ps1.
https://github.com/brandoncomputer/powershell-designer/blob/646557001d30cfc61218bce248d2f27a0388b9bd/Designer.ps1#L393

I've only tested loading and saving forms with simple controls (textbox, button). Not sure if this can have sideeffects on any controls depending on locales. Datepickers maybe?

**Comment by Miwer** [comment]
@loial I am having the same problem. I tried adding the line you suggested in Designer.ps1, but that does not fix it for me.

Running Danish Locale

exception error message:

```
20230613 08:31:30: Cannot convert value "800; 600" to type "System.Int32". Error: "Input string was not in a correct format." At line:1024 char:6 + $n[0] = [math]::round(($n[0]/1) / $tscale) + ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~  + CategoryInfo : InvalidArgument: (:) [], RuntimeException  + FullyQualifiedErrorId : InvalidCastFromStringToInteger  
```

**Comment by Miwer** [comment]
It seems, the trouble is caused by the split commands, only splitting values with a comma delimiter, where other locales may use semicolon as delimiter. Take the example below, Line 303 in Designer.ps1
```
$n = $attrib.Value.split(',')
```
Powershell by default uses regular expression to match the delimiter, so this could be changed to the following, to match both cases of comma, and semicolon delimiters:
```
$n = $attrib.Value.split('[,;]')
```

I found 20 occurrences throughout Designer.ps1 and replaced them all, making sure it made sense, and now I can save my projects. Parameters in the resulting .fbs XML file has the parameters delimited by comma after this. 

Also there are other parameters that are not modified by the script, e.g. AutoScrollMargin, AutoScrollMinSize, Padding, Margin.
Different controls have different properties, so ther might be more, that needs to be added.

As they are not included, I cannot modify them in the designer, or it would save an invalid .fbs file with some parameters retaining the semicolon.

**Comment by brandoncomputer** [comment]
Merged fix from [loial](https://github.com/loial)

**Comment by brandoncomputer** [comment]
Not committed to events.ps1, so change got undone with future versions. Need to reimplement.

**Comment by brandoncomputer** [comment]
(Re) Fixed in 2.6.2
